docs/runtime/bunfig.md

```diff
@@ -457,8 +457,6 @@ editor = "code"
 
 The `bun run` command can be configured under the `[run]` section. These apply to the `bun run` command and the `bun` command when running a file or executable or script.
 
-Currently, `bunfig.toml` isn't always automatically loaded for `bun run` in a local project (it does check for a global `bunfig.toml`), so you might still need to pass `-c` or `-c=bunfig.toml` to use these settings.
-
```

Comment:
this pr does also change this to default import local project `bunfig.toml` as its more expected and inline with `bun install`

did temporally include change that removed `-c`  by itself as a feature, but decided to not pursue as really messed some things up.